components/dashboard/src/components/PillLabel.tsx

```diff
@@ -4,12 +4,14 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-export type PillType = "info" | "warn" | "success";
+export type PillType = "info" | "warn" | "success" | "warnStrong" | "subtle";
 
 const PillClsMap: Record<PillType, string> = {
     info: "bg-blue-50 text-blue-500 dark:bg-blue-500 dark:text-blue-100",
     warn: "bg-orange-100 text-orange-700 dark:bg-orange-600 dark:text-orange-100",
     success: "bg-green-100 text-green-700 dark:bg-green-600 dark:text-green-100",
+    warnStrong: "bg-orange-500 text-white dark:bg-orange-600",
```

Comment:
**thought**: I think we can safely remove the need for the `warnStrong` variant here. Using the warn variant should suffice.